pontoon/base/models.py

```diff
@@ -2245,6 +2245,29 @@ def unchanged(self, locale, project=None):
             )
         )
 
+    def stale(self, locale, project=None):
+        """Return a filter to be used to select entities that have stale translations.
+
+        An entity is considered "stale" if all of its plural forms have translations
+        that have not been updated since before the source string was last updated.
+
+        :arg Locale locale: a Locale object to get translations for
+
+        :returns: a django ORM Q object to use as a filter
+
+        """
+        return Q(
+            pk__in=self.get_filtered_entities(
+                locale,
+                Q(
+                    Q(approved=True) & Q(date__lt=F("entity__date_updated"))
```

Comment:
`approved=True` is probably not right here, since that can be filtered separately.  Should this include an `active=True` filter?  I saw that on the `unchanged` query, but wasn't sure what it meant.